evaluator/src/main/resources/rules/no_gpl_declared.kts

```diff
@@ -2,13 +2,30 @@
 //
 // $ ort evaluate -i scanner/src/funTest/assets/file-counter-expected-output-for-analyzer-result.yml --rules-resource /rules/no_gpl_declared.kts
 
-val pkgWithGpl = ortResult.analyzer?.result?.packages?.filter { (pkg, _) ->
-    pkg.declaredLicenses.any { license ->
-        license.startsWith("GPL")
+// Define a custom rule matcher.
+fun PackageRule.LicenseRule.isGpl() =
+    object : RuleMatcher {
+        override val description = "isGpl($license)"
+
+        override fun matches() = license.contains("GPL")
+    }
+
+// Define the rule set.
+val ruleSet = ruleSet(ortResult) {
+
```

Comment:
Nit (can be done later): Remove superfluous blank line?

Reply:
Done.